solutions/src/main/java/ru/sberbank/school/task06/CollectionUtils.java

```diff
@@ -0,0 +1,137 @@
+package ru.sberbank.school.task06;
+
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+
+public class CollectionUtils {
+
+    /**
+     * Добавление всех элементов одной коллекции в другую.
+     *
+     * @param source      источник
+     * @param destination потребитель
+     */
+    public static <T> void addAll(List<? extends T> source, List<? super T> destination) {
+        destination.addAll(source);
+    }
+
+    /**
+     * Инициализация листа с реализаций java.uXtil.ArrayList.
+     *
+     * @return пустой ArrayList
+     */
+    public static <T> List<T> newArrayList() {
+        return new ArrayList<>();
+    }
+
+    /**
+     * Поиск индекса элемента в списке.
+     *
+     * @param source список, в котором искать элемент
+     * @param o      элемент
+     * @return индекс элемента
+     */
+    public static <T> int indexOf(List<? extends T> source, T o) {
+       return source.indexOf(o);
```

Comment:
&#39;method def&#39; child has incorrect indentation level 7, expected level should be 8.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-211061503>*